tests/test_utils.py

```diff
@@ -201,6 +201,20 @@ def test_uri_decode(self):
             'http://example.com?x=ab%2Bcd%3D42%2C9'
         ) == 'http://example.com?x=ab+cd=42,9'
 
+    def test_uri_decode_unquote_plus(self):
```

Comment:
It would be nice to see pytest parametrize here instead of individual `assert`, but that is a nit-pick beyond nitpicks!

Reply:
@nZac indeed :+1:  I was considering that route, but I wanted to include tests for both specifying the kwarg parameter, and omitting it, which made parametrization somewhat more cumbersome.

But maybe I could fully parametrize `*args` and `**kwargs`? It is easy to change, let me know if you think it is worth pursuing.

Reply:
That's fine. I am good without it.